### PR TITLE
[browser] Add ability of using cutout API. Contributes to JB#56709

### PR DESCRIPTION
**Pull request by atatarov**

### CODE REVIEW

**Comment by rainemak** [comment]
There's nothing OMP specific in the commit per se. Please drop "from omp side" from the commit message and merge commit message.

File: apps/shared/BrowserWindow.qml
Comment (rainemak) [inline apps/shared/BrowserWindow.qml:64]: Where is displayMode and ApplicationDisplayMode defined?

**Review by llewelld — APPROVED** [review]
I had a quick test and didn't notice any issues with this. So looks good to me. But I'd also await @rainemak's approval before merging.

**Review by okodron — APPROVED** [review]

File: apps/shared/BrowserWindow.qml
Comment (rainemak) [reply apps/shared/BrowserWindow.qml:64]: For adding a comment that this is coming from Silica.

**Pull request merged**
